packages/esm-service-queues-app/src/createDashboardLink.component.tsx

```diff
@@ -2,18 +2,21 @@ import React, { useMemo } from 'react';
 import classNames from 'classnames';
 import { BrowserRouter, useLocation } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { ConfigurableLink } from '@openmrs/esm-framework';
-
+import { ConfigurableLink, useConfig } from '@openmrs/esm-framework';
+import { Button } from '@carbon/react';
+import styles from './root.scss';
 export interface DashboardLinkConfig {
   name: string;
   title: string;
+  renderIcon: React.ComponentType<any>;
 }
 
 function DashboardExtension({ dashboardLinkConfig }: { dashboardLinkConfig: DashboardLinkConfig }) {
```

Comment:
Maybe we should turn this into a component? Almost nothing here (except the text `serviceQueues` is app-specific.

Reply:
https://openmrs.atlassian.net/browse/O3-3506